### ISSUE TITLE
Some domain name have no associated topic

### ISSUE THREAD

**Issue by francoishumanoid** [patcg-individual-drafts/topics#264]
We tried to see which topics are associated with our websites, and see that one of our domain has no topics associated. We have topics associated with frandroid.com and madmoizelle.com but numerama.com is not returning any associated topic.

How can we associate topics with numerama.com ?

Thanks

**Comment by leeronisrael** [comment]
We do not offer a mechanism for publishers to modify the topics assigned to their domains. Our rationale is that the misclassified sites that are are not directly harmed. This is because a site’s contextual information will always be available for auctions on their site, which would provide comparable information to the correct topic.

Further, we expect that the classifier will improve over time. The updated classifier being shipped as part of the taxonomy v2 [ramp up](https://github.com/patcg-individual-drafts/topics/issues/236) assigns [numerama.com](http://numerama.com/) two topics: “Computers & electronics” and “News”. You can observe this directly by taking the following steps:

1. Navigate to chrome://flags and enable “Privacy Sandbox Ads APIs”
2. Enable taxonomy V2 via a command line flag `--enable-features=BrowsingTopicsParameters:taxonomy_version/2` ([docs](https://developer.chrome.com/docs/web-platform/chrome-flags/#command-line-flags))
3. Navigate to chrome://topics-internals and select the “Classifier” tab
4. Enter http://numerama.com/ in the field and select “Classify”

**Comment by patmmccann** [comment]
this feels like a dupe of https://github.com/patcg-individual-drafts/topics/issues/224#issuecomment-1679310822

**Comment by leeronisrael** [comment]
We do not expect there to be a negative impact to individual sites that are misclassified or lack a classification.  A key benefit of Topics is surfacing interests that expand beyond a site’s own contextual information.  A site’s own contextual information can continue to be accessible, independently of the Topics API (as it is today). While there is not a way for an individual site to request specific topics to be associated with their site, we do expect the classifier to improve over time.  

The latest classifier is rolling out following the same [ramp up as the updated](https://groups.google.com/u/0/a/chromium.org/g/topics-api-announce/c/iNYk69wKnJs) taxonomy  This updated classifier assigns [numerama.com](http://numerama.com/) two topics: “Computers & electronics” and “News”. You can observe this directly by taking the following steps:

1. Navigate to chrome://flags and enable “Privacy Sandbox Ads APIs”
2. Enable taxonomy V2 via a command line flag `--enable-features=BrowsingTopicsParameters:taxonomy_version/2` ([docs](https://developer.chrome.com/docs/web-platform/chrome-flags/#command-line-flags))
3. Navigate to chrome://topics-internals and select the “Classifier” tab
4. Enter http://numerama.com/ in the field and select “Classify”

**Comment by patmmccann** [comment]
> We do not expect there to be a negative impact to individual sites that are misclassified or lack a classification. 

I disagree; sites that lack classifications may be excluded from high-value topics sharing networks as known free riders.

**Comment by francoishumanoid** [comment]
@leeronisrael thanks for the reply.

**Comment by dmarti** [comment]
@patmmccann In some cases, a domain could be correctly classified with a high-value topic at the time the domain is bought, but misclassified or unclassified by the version of the classifier in use at the time the site goes live and/or is looking for a topics sharing network to join.

**Comment by ghasemiradarash-web** [comment]
> > ما انتظار نداریم که تأثیر منفی بر سایت‌های منفردی که به اشتباه طبقه‌بندی شده‌اند یا فاقد طبقه‌بندی هستند، داشته باشیم.
> 
> من مخالفم؛ سایت‌هایی که فاقد طبقه‌بندی هستند، ممکن است به عنوان سایت‌های مفت‌خور از شبکه‌های اشتراک‌گذاری موضوعات ارزشمند حذف شوند.

بله ممکنه دربرخی موارد دیده شده خطای سیستمی باعثحذف اشتباه یا تایید عضویت مخرب شده
